src/inference/tests/unit/query_model_test.cpp

```diff
@@ -387,7 +387,7 @@ TEST_F(GetSupportedNodesTest, ShapeOfNonConstantNode) {
     {
         auto param = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, m_shape);
         param->set_friendly_name("input");
-        auto slope_compressed = ov::op::v0::Constant::create(ov::element::f16, ov::Shape{}, {-2.f});
```

Comment:
why do we need to change tests?
I suppose CF should know how to fold f16 constants using convert to f32